src/pages/receipts/packagesInTransit.js

```diff
@@ -0,0 +1,391 @@
+            } = parsedModalData;
+
+            let modalBody = `<div class="container-fluid">
+            <div class="row">
+                <div class="col-md-4">
+                    <p style="font-size:1.3rem;"><strong>Shipping Manifest</strong></p>
+                </div>
+                <div class="col-md-4 ml-auto">
+                    <p><strong>Site:</strong> ${site ? site : "N/A"} </p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-4">
+                    <p><strong>Shipped Date and Time:</strong> ${date ? convertTime(date) : "N/A"}</p>
+                </div>
+                <div class="col-md-4 ml-auto">
+                    <p><strong>Location:</strong> ${location ? location : "N/A"}</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-4">
+                    <p><strong>Sender:</strong><br/>${namesArr[index] ? namesArr[index].toString().replaceAll("," ,`<br/>`) : "N/A"}</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="table-responsive">
+                    <table class="table table-striped">
+                        <thead>
+                        <tr>
+                        <th class="sticky-row" style="background-color: #f7f7f7; text-align:center;" scope="col-">Box Number</th>
+                        <th class="sticky-row" style="background-color: #f7f7f7; text-align:center;" scope="col">Specimen Bag ID</th>
+                        <th class="sticky-row" style="background-color: #f7f7f7; text-align:center;" scope="col">Full Specimen ID</th>
+                        <th class="sticky-row" style="background-color: #f7f7f7; text-align:center;" scope="col">Scanned By</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        ${addManifestTableRows(boxNumber, bagIdArr, index, groupSamples, groupScannedBy)}
+                    </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>`;
+        manifestModalBodyEl.innerHTML = modalBody;
+        });
+    });
+};
+
+// Convert utc in Seconds to readable Date
+const convertTime = (time) => {
+    if (!time) {
+        return "";
+    }
+    let utcSeconds = time;
+    const myDate = new Date(utcSeconds);
+    return myDate.toLocaleString("en-us", {
+        year: "numeric",
+        month: "numeric",
+        day: "numeric",
+        hour: "2-digit",
+        minute: "2-digit",
+    });
+};
+
+// Return an array of an item of grouped bags from GET request***
+const groupAllBags = (allBoxes) => {
+    const arrBoxes = [];
+    // Object.keys --> Copies Keys and stores into array
+    // If Key(bags) has a length push bag of objects, else an empty {}
+    allBoxes.forEach((box) => {
+        Object.keys(box.bags).length ? arrBoxes.push(box.bags) : arrBoxes.push(box.bags);
```

Comment:
In the  else condition is arrBoxes.push(box.bags) pushing {} to the bag?

Reply:
Yeah the else path after the colon operator is pushing an empty object to the array. 
I thought about adding && operators instead:
<img width="501" alt="Screen Shot 2021-09-30 at 11 17 37 AM" src="https://user-images.githubusercontent.com/33293205/135483588-90c6b4bf-c79d-43e8-bdb0-b28544e2396c.png">

I think I need the empty object to be pushed because the other functions rely on the order for bagsArr. Also, && would return false if there is an empty length resulting in an omitted push and shorter array length.

bagsArr with && operator:

<img width="684" alt="Screen Shot 2021-09-30 at 11 11 45 AM" src="https://user-images.githubusercontent.com/33293205/135484289-245286ab-bce8-42cb-9dbd-428e1fc59b31.png">

bagsArr with ternary operator:

<img width="554" alt="Screen Shot 2021-09-30 at 11 05 45 AM" src="https://user-images.githubusercontent.com/33293205/135484235-74709b87-5817-44c4-b327-542315529d56.png">